CHANGELOG.md

```diff
@@ -2,6 +2,77 @@
 
 Nothing Yet!
 
+# Version 0.26.0 (20XX-XX-XX)
+
+It's been slightly longer than usual since our last release, and now we're back with a slightly larger than usual release! This version brings several major new features and improvements, including the long-requested Rust cross-compilation feature and support for a few different Rust dependency version tracking formats.
+
+## Builtin Rust cross-compilation support
+
+You've all asked for it, and it's finally here! Previously, we only supported Rust cross-compilation on macOS. With this release, we've extended Rust cross-compilation support to Linux (using [cargo-zigbuild](https://github.com/rust-cross/cargo-zigbuild) and Windows (using [cargo-xwin](https://github.com/rust-cross/cargo-xwin).
+
+We're also making use of this feature ourselves: we now build our aarch64 Linux binaries using this new tooling.
+
+(Note: for technical reasons, this feature and cargo-auditable are currently mutually exclusive. Users can only enable one or the other.)
+
+- impl @fasterthanlime [Add cross-compilation support via cargo-zigbuild and cargo-xwin](https://github.com/axodotdev/cargo-dist/pull/1529)
+
+## Checksum verifications in shell installers
+
+While we've always generated checksum information for binary tarballs/ZIPs, we only actually validated those checksums in the Homebrew installer. That changes with this release: we now embed checksum information into the shell installer and validate the tarball before unpacking it.
+
+- impl @fasterthanlime [Verify checksums in install.sh](https://github.com/axodotdev/cargo-dist/pull/1497)
+
+## cargo-auditable support
+
+We've added integrated support for the Rust Secure Code Working Group's [cargo-auditable](https://github.com/rust-secure-code/cargo-auditable), which embeds dependency information in your Rust binaries and makes it possible for users to check your binaries for the full dependency tree they were built from with their precise versions. For more information, see [our docs](https://opensource.axo.dev/cargo-dist/book/reference/config.html#cargo-auditable) and the [docs for the cargo-audit tool](https://github.com/rustsec/rustsec/blob/main/cargo-audit/README.md).
+
+(Note: for technical reasons, this feature and cross-compilation are currently mutually exclusive. Users can only enable one or the other.)
```

Comment:
this is interesting... should we file an issue to eliminate this?

Reply:
Unfortunately, I don't think we can. All three of `cargo-auditable`, `cargo-xwin` and `cargo-zigbuild` are implemented as cargo subcommands that wrap the real cargo builds. It's not possible to run more than one of them in the same build.

Reply:
IMO this is not strictly true, see https://github.com/rust-cross/cargo-zigbuild/issues/192

Reply:
Yep, been looking into it - #1608. Looking like the zigbuild stuff may be incompatible for other reasons, at least in testing so far.